File: [프로그래머스] 코딩테스트 입문_영어가 싫어요.py
def solution(numbers):
    alpha = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
    for i, j in zip(alpha, range(10)):
        numbers = numbers.replace(i, str(j))
    return int(numbers)

---------------------------

def solution(numbers):
    answer = 0
    num = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
    for i in num:
        if i in numbers:
            numbers = numbers.replace(i, str(num.index(i)))
    return int(numbers)

File: [프로그래머스] 코딩테스트 입문_영어가 싫어요.js
function solution(numbers) {
    let dic = {0:"zero", 1:"one", 2:"two", 3:"three", 4:"four", 5:"five", 6:"six", 7:"seven", 8:"eight", 9:"nine"};
    for(let i = 0; i <= 9; i++){
        if(numbers.includes(dic[i])){
            numbers = numbers.replaceAll(dic[i], i);
        }
    }
    return Number(numbers);
}